bin/sl-deploy.js

```diff
@@ -114,24 +123,36 @@ if (!local)
     }
     debug('Connecting to %s via %s', baseURL, url);
     deploy(
-      workingDir,
-      url,
-      serviceName,
-      branchOrPack,
+      {
+        workingDir: workingDir,
+        baseURL: baseURL,
+        serviceName: serviceName,
+        branchOrPack: branchOrPack,
+        clusterSize: clusterSize,
+      },
       exit
     );
   });
 else
-  deploy.local(baseURL, serviceName, branchOrPack, exit);
+  deploy.local(
+    {
+      baseURL: baseURL,
+      serviceName: serviceName,
+      branchOrPack: branchOrPack,
+      clusterSize: clusterSize,
+    },
+    exit);
 
 function exit(err, service) {
   var svc = service ? (service.name || service.id) : serviceName;
   if (err) {
-    console.error('Failed to deploy `%s` as `%s` to `%s` via `%s`',
-                  branchOrPack, svc, baseURL, exit.url);
+    console.error(
+        'Failed to deploy `%s` as `%s` to `%s` via `%s`, with cluster size `%d`',
```

Comment:
I appreciate the effort to provide rich feedback, but since clusterSize will become optional, and is unlikely to be the cause of a deployment failure, I would remove it from these two console.error statements.